src/control/cmd/dmg/pool.go

```diff
@@ -112,6 +113,21 @@ func (d *PoolOverwriteACLCmd) Execute(args []string) error {
 	return poolOverwriteACL(d.log, d.conns, d.UUID, d.ACLFile)
 }
 
+// PoolUpdateACLCmd represents the command to update the Access Control List of
+// a DAOS pool.
+type PoolUpdateACLCmd struct {
+	logCmd
+	connectedCmd
+	UUID    string `long:"pool" required:"1" description:"UUID of DAOS pool"`
+	ACLFile string `short:"a" long:"acl-file" required:"0" description:"Path for new Access Control List file"`
+	Entry   string `short:"e" long:"entry" required:"0" description:"Single Access Control Entry to add or update"`
+}
+
+// Execute is run when the PoolUpdateACLCmd subcommand is activated
+func (d *PoolUpdateACLCmd) Execute(args []string) error {
+	return poolUpdateACL(d.log, d.conns, d.UUID, d.ACLFile, d.Entry)
```

Comment:
I know this has been done historically but probably don't need to separate this into a separate function, can just put the logic in the Execute method

Reply:
I think I'd like to address this in a follow-on PR, assuming nothing else needs to be changed. I agree, I was kind of curious why it was broken out.